### ISSUE TITLE
sync issue with items with :SCHEDULED: or :DEADLINE: keywords

### ISSUE THREAD

**Issue by oguzcanoguz** [dengste/org-caldav#33]
If a task has only a date stamp in org then the sync seems to work fine. However, if I have either of the useful :SCHEDULED: or :DEADLINE: keywords in org before the date stamp, then, in the first sync, these events go into the calendar but in the second sync they get removed from the calendar with the claim that they got deleted in org. I guess org-caldav somehow cannot find these kind of events in org after the first sync.

**Comment by IvanMalison** [comment]
close?

**Comment by emelin** [comment]
No, please, there actually is an issue! I observe the following behaviour since several weeks with growing discomfort:

Only by the second sync (exactly as described above) the SCHEDULED time gets converted to an active timestamp and a freshly created property drawer with a new id is inserted above that newly created active timestamp.

My settings (perfectly legitimate) are:

(setq org-icalendar-include-todo 'all)
(setq org-icalendar-categorie '(all-tags category todo-state))
(setq org-icalendar-use-deadline '(event-if-not-todo todo-due))
(setq org-icalendar-use-scheduled '(todo-start event-if-todo event-if-not-todo))
(setq org-icalendar-with-timestamps 'active)
(setq org-icalendar-alarm-time 30)

I follow the org-mode master-branch. Current org-version is:

 Org-mode version 8.3beta (release_8.3beta-835-g13938b
